Product-Designer.md

```diff
@@ -0,0 +1,55 @@
+# Work Sample for Product Designer
+
+You are tasked with designing the user interface for a climate risk dashboard web app. Your goal is to create a cohesive and user-friendly design that allows users to interact with the provided dataset and gain insights into climate change and its risk implications.
+
+You may refer to the [Climate Risk Rating dataset](https://docs.google.com/spreadsheets/d/1Y_yiT-_7IimioBvcqiCPwLzTLazfdRyzZ4k3cpQXiAw) to assist with your solutions.
+
+## Problem 1: Information Architecture and Wireframing
```

Comment:
If my understanding is correct

problem 1 asks the candidate to design everything at a high level
then problems 2 and 3 asks for a more detailed design of the map component
and problem 4 asks how they generalize the map style to other pages

Reply:
Yes, pretty much!

Reply:
In [2792e01](https://github.com/RiskThinking/work-samples/pull/5/commits/2792e014cb505f3331807400f155e6f9d57d7629) I consolidated 2-3